_includes/sidebar.html

```diff
@@ -10,113 +10,113 @@
   {% for category in site.data.documentsupdated %} {% if forloop.last == true %}
   <li class="folder topfolder lowerimage">
     <a class="highlightlink" href="">
-      <span class="sidebarbutton"><span></span></span>
-      <span> {{ category.categoryname }} </span></a
-    >
+      <span class="sidebarbutton">
+        <span></span>
+      </span>
+      <span> {{ category.categoryname }}</span>
+    </a>
     <ul>
       {% for document in category.documents %} {% if document.pages %}
       <li class="folder innerfolder sidebarcategory">
-        <a class="highlightlink" href=""
-          ><span class="sidebarbutton innerArrowAdjust"><span></span></span
-          ><span> {{ document.documentname }}</span></a
-        >
+        <a class="highlightlink" href="">
+          <span class="sidebarbutton innerArrowAdjust">
+            <span></span>
+          </span>
+          <span> {{ document.documentname }}</span>
+        </a>
         <ul>
           {% for page in document.pages %} {% if page.subpages %}
           <li class="folder innerfolder subfolder">
-            <a href=""
-              ><span class="sidebarbutton innerArrowAdjust"> <span></span></span
-              ><span>{{ page.pagename }}</span></a
-            >
+            <a href="">
+              <span class="sidebarbutton innerArrowAdjust">
+                <span></span>
+              </span>
+              <span>{{ page.pagename }}</span>
+            </a>
             <ul class="innerList">
               {% for subpage in page.subpages %}
               <li class="innerpageitem">
                 <a
                   onclick="linkclick(event,this)"
                   class="innerlink"
                   href="/{{ document.documentname | append: '/' | append: page.pagename | append: '/' |append: subpage.subpagename | slugify }}.html"
-                  >{{ subpage.subpagename }}</a
-                >
+                  >{{ subpage.subpagename }}
+                </a>
```

Comment:
Please undo or propose a more symmetric style than having the content follow the start tag right away, but not precede the end tag.